src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEnum.cs

```diff
@@ -22,17 +22,16 @@
 
 namespace Tizen.NUI.BaseComponents
 {
-    public partial class ReadingInfoTypes
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    [Flags]
+    public enum AccessibilityReadingInfoType : int
```

Comment:
please check the warning of "Flags enums should have plural names"
it seems that `AccessibilityReadingInfoType` need be changed as `AccessibilityReadingInfoTypes`

Reply:
Done